lib/collection/widgets/first_pass_screen.dart

```diff
@@ -60,7 +60,7 @@ class FirstPassScreen extends StatelessWidget {
                 children: <Widget>[
                   FloatingActionButton.extended(
                       heroTag: null,
-                      label: const Text("New Food Item"),
```

Comment:
I was under the impression that food item was a specific word used by ICRISAT for this purpose - if we're moving away from using food item then internal variables ought to be updated too? Theres at least one other place in the UI where 'food item' is still used - on the probes screen when accessing via collection

Reply:
I wasn't aware that `Food Item` was a specific word used by ICRISAT — if I remember correctly, I just first used it in Collection because of a lack of better term for the given object. However, the name was inspired to keep the backend expressive, not with the UI in mind.

Again, this just comes down to preference — I personally imagine a user would find it more natural to "add new food" compared to "add new food _item_", and I think it is perfectly alright to keep the user-visible-button name different to the "backend" implementation naming, since the event the button sends is still clear in that it is a FoodItem.

tl;dr we can ask ICRISAT about it.